libs/feature/teaching/src/lib/course/course-learnpath-editor/skill-lesson-linker.tsx

```diff
@@ -0,0 +1,163 @@
+import { Mesh, SkillFormModel } from "@self-learning/types";
+import { SelectSkillDialog } from "../../skills/skill-dialog/select-skill-dialog";
+import { IconButton } from "@self-learning/ui/common";
+import { PlusIcon } from "@heroicons/react/24/solid";
+import { LessonSelector, LessonSummary } from "../course-content-editor/dialogs/lesson-selector";
+import { useState } from "react";
+
+export function SkillLessonLinker({
+	repositoryId,
+	onClick: submitMesh,
```

Comment:
das  renaming verwirrt hier etwas. Kann man so machen, es macht insbesondere in der methode klar, wofür onClick verwindert wird. 

Ist das aber auch von außen klar wenn die komponente aufgerufen wird? 
Mir ist nicht ganz klar wofür SkillLessonLinker ein onClick hat. (bei einem button ist das zB ziemlich klar).